modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -392,6 +392,52 @@ public void testPostObjectDefinition() throws Exception {
 
 		assertEquals(postObjectDefinition, randomObjectDefinition);
 		assertValid(postObjectDefinition);
+
+		// Add Object definition with a label in a language different from the
+		// site language
+
+		ObjectDefinition objectDefinition = randomObjectDefinition();
+
+		objectDefinition.setDefaultLanguageId("de_DE");
+		objectDefinition.setObjectFields(
+			new ObjectField[] {
+				new ObjectField() {
+					{
+						businessType = BusinessType.TEXT;
+						DBType = ObjectField.DBType.create("String");
+						indexed = false;
+						indexedAsKeyword = false;
+						label = Collections.singletonMap("de_DE", "Random");
+						localized = !objectDefinition.getSystem();
+						name = StringUtil.randomId();
+						readOnly = ReadOnly.FALSE;
+						required = false;
+						system = false;
+					}
+				}
+			});
+
+		ObjectDefinition objectDefinitionWithDifferentLabels =
```

Comment:
Can you rename this to `multipleLanguagesObjectDefinition`? The other object definitions in this test follow the pattern of `XXObjectDefinition`